ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -159,36 +159,45 @@ public struct AvatarGroup: View {
         let overflowCount: Int = (avatars.count > maxDisplayedAvatars ? avatars.count - maxDisplayedAvatars : 0) + state.overflowCount
 
         let interspace: CGFloat = tokens.interspace
-        let size: CGFloat = tokens.size.size
+        let imageSize: CGFloat = tokens.size.size
         let ringOuterGap: CGFloat = tokens.ringOuterGap
+        let ringGapOffset: CGFloat = ringOuterGap * 2
         let ringOffset: CGFloat = tokens.ringThickness + tokens.ringInnerGap + tokens.ringOuterGap
-        let x: CGFloat = size + tokens.interspace - tokens.ringThickness
         HStack(spacing: 0) {
             ForEach(0 ..< maxDisplayedAvatars, id: \.self) { index in
                 // If the avatar is part of Stack style and is not the last avatar in the sequence, create a cutout
-                let needsCutout = tokens.style == .stack && (overflowCount > 0 || index + 1 < maxDisplayedAvatars)
                 let avatar = avatars[index]
+                let avatarView = Avatar(avatar)
+                let avatarSize = avatarView.size()
+                let nextAvatarSize = Avatar(avatars[index + 1]).size()
```

Comment:
Two thoughts about this line:
1. Does this not crash/throw when `index + 1 == maxAvatars`? (That is, when `index` is referring to the last avatar)
2. I'm a little bummed that we need to create an `Avatar` for the `index + 1` state that'll just get thrown away. Maybe we could move the `size` function to the `state` object instead?

Reply:
There is already a size call in the state object, but what do you think of the update?

Reply:
Agree with @mischreiber on moving the size func to the state object. (Didn't see the comment about the size func before I added that other comment about it.)

Reply:
@sophialee0416 I think frameSize could be a good option for the name of the func. Thoughts?